feeds/blocks.go

```diff
@@ -243,7 +243,23 @@ func (bf *blockFeed) forEachBlock() error {
 			continue
 		}
 
-		evt := &domain.BlockEvent{EventType: domain.EventTypeBlock, Block: block, ChainID: bf.chainID, Traces: traces, Logs: logs}
+		blockTs, err := block.GetTimestamp()
+		if err != nil || blockTs == nil {
+			logger.WithError(err).Error("failed to get block timestamp")
```

Comment:
`error=<nil> msg=failed to get block timestamp` should we expect a message like this any time?

Reply:
:+1: removing check